README.md

```diff
@@ -52,18 +52,26 @@ AppRegistry.registerComponent('NfcManagerDev', () => App);
 ```
 
 ## API
-This library provide a default export `NfcManager` and 2 named exports `ByteParser` and `NdefParser`, like this:
+This library provide a default export `NfcManager` and 3 named exports `Ndef`, `NfcTech` and `ByteParser`, like this:
 ```javascript
-import NfcManager, {ByteParser, Ndef, NdefParser} from 'react-native-nfc-manager'
+import NfcManager, {Ndef, NfcTech, ByteParser} from 'react-native-nfc-manager'
 ```
 
 All methods in `NfcManager` return a `Promise` object and are resolved to different types of data according to individual API.
 
+* `Ndef` is an utility module to encode and decode some well-known NDEF format.
 * `ByteParser` is an utility module to encode and decode byte[] arrays (used in Mifare Classic technology).
+* `NfcTech` contains predefined constants for specific NFC technologies, which include `NfcA`, `NfcB`, `NfcF`, `NfcV`, `IsoDep` and `MifareClassic`.
+    * These constants should be used with `requestTechnology` (Android Only) to obtain a NFC technology handle, and use it to perform technology specific operations.
 
-* `Ndef` is an utility module to encode and decode some well-known NDEF format.
+Detailed API document is grouped into 6 parts:
```

Comment:
Unsure what you mean with this phrase. Do you mean "The API documentation is grouped into 6 parts"?

Reply:
Good catch, thanks!